lib/process-services-cloud/src/lib/task/task-list/components/service-task-list-cloud.component.spec.ts

```diff
@@ -396,42 +365,35 @@ describe('ServiceTaskListCloudComponent', () => {
             expect(componentCustom.taskList.columns.length).toEqual(2);
         });
 
-        it('it should show copy tooltip when key is present in data-colunn', fakeAsync(() => {
+        it('it should show copy tooltip when key is present in data-column', () => {
+            customCopyComponent.taskList.reload();
             copyFixture.detectChanges();
-            const appName = new SimpleChange(null, 'FAKE-APP-NAME', true);
-            copyFixture.whenStable().then(() => {
-                copyFixture.detectChanges();
-                const spanHTMLElement = element.querySelector<HTMLInputElement>('span[title="04fdf69f-4ddd-48ab-9563-da776c9b163c"]');
-                spanHTMLElement.dispatchEvent(new Event('mouseenter'));
-                copyFixture.detectChanges();
-                expect(copyFixture.debugElement.nativeElement.querySelector('.adf-copy-tooltip')).not.toBeNull();
-            });
-            customCopyComponent.taskList.appName = appName.currentValue;
-            customCopyComponent.taskList.ngOnChanges({ appName });
+
+            copyFixture.debugElement
+                    .query(By.css('span[title="04fdf69f-4ddd-48ab-9563-da776c9b163c"]'))
+                    .triggerEventHandler('mouseenter');
+
             copyFixture.detectChanges();
-        }));
+            expect(copyFixture.debugElement.query(By.css('.adf-copy-tooltip'))).not.toBeNull();
+        });
 
-        it('it should not show copy tooltip when key is not present in data-column', (done) => {
-            const appName = new SimpleChange(null, 'FAKE-APP-NAME', true);
+        it('it should not show copy tooltip when key is not present in data-column', () => {
             customCopyComponent.taskList.success.subscribe(() => {
```

Comment:
again, this might be true now, but not always. Without the done function unit test won't wait for the subscription to be done and soon we will have errors like : Spec ended without expectation. IF you think the test was wrong please refactor it in a way that makes more sense